docker/pulsar-all/Dockerfile

```diff
@@ -19,6 +19,7 @@
 
 ARG PULSAR_IMAGE
 FROM busybox as pulsar-all
+ENV PTHREAD_STACK_MIN 2097152
```

Comment:
This is a multi-stage image. The `ENV` is only effective in the final image if it's defined in the last stage.